src/frontend/src/flows/pin/pinInfo.ts

```diff
@@ -0,0 +1,86 @@
+import { mainWindow } from "$src/components/mainWindow";
+import { pinStepper } from "$src/flows/pin/stepper";
+import { I18n } from "$src/i18n";
+import { mount, renderPage } from "$src/utils/lit-html";
+import { TemplateResult, html } from "lit-html";
+
+import copyJson from "./pinInfo.json";
+
+const pinInfoTemplate = ({
+  i18n,
+  onContinue,
+  cancel,
+  scrollToTop = false,
+}: {
+  onContinue: () => void;
+  i18n: I18n;
+  cancel: () => void;
+  /* put the page into view */
+  scrollToTop?: boolean;
+}): TemplateResult => {
+  const copy = i18n.i18n(copyJson);
+
+  const slot = html`
+    ${pinStepper({ current: "set" })}
+    <hgroup ${scrollToTop ? mount(() => window.scrollTo(0, 0)) : undefined}>
```

Comment:
I guess copy / paste from somewhere else right?

Reply:
Yes, this page was made to be basically the same as the one defined in `passkey.ts` (which is the passkey info page).

Reply:
Ok